roles/kubernetes/master/templates/kubeadm-config.v1alpha2.yaml.j2

```diff
@@ -24,13 +24,17 @@ kubernetesVersion: {{ kube_version }}
 {% if cloud_provider is defined and cloud_provider != "gce" %}
 cloudProvider: {{ cloud_provider }}
 {% endif %}
+{% if kube_proxy_mode == "none" %}
```

Comment:
Should this be added to the v1alpha1 template too?

Reply:
Done